app/src/main/java/dev/arkbuilders/arkmemo/graphics/SVG.kt

```diff
@@ -33,12 +35,16 @@ class SVG {
         commandsArray.add(command)
     }
 
-    fun moveTo(x: Float, y: Float) = "$SPACE$MOVE_TO$SPACE$x,$y"
+    fun addPath(path: DrawPath) {
+        paths.add(path)
+    }
+
+    fun moveTo(x: Float, y: Float) = "$MOVE_TO$SPACE$x$SPACE$y"
 
-    fun relativeLineTo(x: Float, y: Float) = "$SPACE$ABS_LINE_TO$SPACE$x,$y"
+    fun relativeLineTo(x: Float, y: Float) = "$ABS_LINE_TO$SPACE$x$SPACE$y"
 
     fun quadraticBezierTo(x1: Float, y1: Float, x2: Float, y2: Float) =
-        "$SPACE$ABS_QUAD_TO$SPACE$x1,$y1$SPACE$x2,$y2"
+        "$ABS_QUAD_TO$SPACE$x1$SPACE$y1$SPACE$x2$SPACE$y2"
```

Comment:
I don't think `$SPACE` constant is really useful. It's more difficult to read with it.

Reply:
But in svg the coordinates should be separated by either a comma or space, not sure if there is another delimiter. So we will use comma to separate commands, coordinates will need another delimiter to avoid messing the data, the option we have is space

Reply:
>  Or you mean reading the code, right? :)

Yeah.. That's the case when following best practices reduces readability/maintainability. Plain `"$ABS_QUAD_TO $x1 $y1 $x2 $y2"` is easier to read and understand.

Reply:
>  Or you mean reading the code, right? :)

Yeah.. That's the case when following best practices reduces readability/maintainability. Plain `"$ABS_QUAD_TO $x1 $y1 $x2 $y2"` is easier to read, understand, maintain etc.

Reply:
I get it, looks cool indeed